persistence/persistence/src/main/java/one/microstream/persistence/types/PersistenceTypeHandlerManager.java

```diff
@@ -452,17 +460,20 @@ public PersistenceTypeDictionary typeDictionary()
 			return this.ensureLegacyTypeHandler(typeDefinition, runtimeTypeHandler);
 		}
 		
-		private <T> PersistenceLegacyTypeHandler<D, T> ensureLegacyTypeHandler(
+		@Override
+		public <T> PersistenceLegacyTypeHandler<D, ? super T> ensureLegacyTypeHandler(
 			final PersistenceTypeDefinition    legacyTypeDefinition,
-			final PersistenceTypeHandler<D, T> currentTypeHandler
+			final PersistenceTypeHandler<D, ? super T> currentTypeHandler
 		)
 		{
-			final PersistenceLegacyTypeHandler<D, T> legacyTypeHandler = this.legacyTypeMapper.ensureLegacyTypeHandler(
+			final PersistenceLegacyTypeHandler<D, ? super T> legacyTypeHandler = this.legacyTypeMapper.ensureLegacyTypeHandler(
 				legacyTypeDefinition,
 				currentTypeHandler
 			);
 			this.registerLegacyTypeHandler(legacyTypeHandler);
 			
+			XDebug.println("registered legacy type handler for " + legacyTypeHandler.typeName() + " " + legacyTypeHandler.typeId());
```

Comment:
XDebug in production code? Logger or?

Reply:
Since logging is in master this is not used anymore.

Reply:
replaced with logging